lib/vscode/src/vs/code/browser/workbench/workbench.ts

```diff
@@ -453,6 +453,14 @@ class WindowIndicator implements IWindowIndicator {
 	// Workspace Provider
 	const workspaceProvider = new WorkspaceProvider(workspace, payload);
 
+	// NOTE@coder: copied from upstream, modified to use our repo
+	// Home Indicator
+	const homeIndicator: IHomeIndicator = {
```

Comment:
Haha of course it came back. Let's not provide a default for the home so we don't have to re-open https://github.com/cdr/code-server/issues/2742.

Reply:
Lol! Agreed

Reply:
I mean it "came back" in the code sense but I can't find it anywhere on my local build, so... :p
I just put the code there for it to compile, lol.

Reply:
Haha fair fair. I believe we can just not provide it and `create` will still be happy.

Reply:
Ooh, that worked!